parachain/test-parachains/adder/collator/src/main.rs

```diff
@@ -37,7 +38,7 @@ fn main() -> Result<()> {
 			let collator = Collator::new();
 			println!("0x{:?}", HexDisplay::from(&collator.genesis_head()));
 
-			Ok(())
+			Ok::<_, Error>(())
```

Comment:
I wonder why this type annotation is required here, but not below in `cli::Subcommand::ExportGenesisWasm` branch, and why at all.

Reply:
The return type could technically by anything that impl `From<..>` error and that way it's tied down in the first branch.